### DISCUSSION TITLE
Zeek SSL notices

### DISCUSSION THREAD

**Discussion by greatapoc** [Security-Onion-Solutions/securityonion#1877] [Unsupported Versions] [answered]
Not sure if this is a question better posed to a Zeek discussion board. I'm just looking through all the Zeek notices I'm receiving on my network for SSL related alerts.

In the past 24 hours I have 34,493 SSL certificate validation failed with (self signed certificate in certificate chain) and 1,855 SSL certificate validation failed with (unable to get local issuer certificate). I'm not sure if these are appearing due to us running SSL decryption on our Palo Alto firewall therefore the cert chains will always contain the certificate for our firewall.

A selection of the notices for SSL certificate validation failed with (self signed certificate in certificate chain) show a whole bunch of perfectly normal looking certificates much the same as the "unable to get local issuer certificate" entries. For example:
CN=*.events.data.microsoft.com,OU=WSE,O=Microsoft,L=Redmond,ST=WA,C=US
CN=settings-win.data.microsoft.com,OU=WSE,O=Microsoft,L=Redmond,ST=WA,C=US
C=US,ST=California,O=Apple Inc.,CN=fmip.icloud.com
CN=*.teamviewer.com
CN=adobe.io,OU=IT,O=Adobe Systems Incorporated,L=San Jose,ST=California,C=US

Is there anything I can do to filter these out perhaps?

**Comment by shipler** [comment]
I think that it's more ZEEK related subject - 

> https://docs.zeek.org/en/current/scripts/policy/protocols/ssl/known-certs.zeek.html

**Reply by presianbg** [reply]
https://gist.github.com/srozb/d1e5770e4fe9f0f60a4fca4a57270da6

**Comment by weslambert** [comment, marked as answer]
> Not sure if this is a question better posed to a Zeek discussion board. I'm just looking through all the Zeek notices I'm receiving on my network for SSL related alerts.
> 
> In the past 24 hours I have 34,493 SSL certificate validation failed with (self signed certificate in certificate chain) and 1,855 SSL certificate validation failed with (unable to get local issuer certificate). I'm not sure if these are appearing due to us running SSL decryption on our Palo Alto firewall therefore the cert chains will always contain the certificate for our firewall.
> 
> A selection of the notices for SSL certificate validation failed with (self signed certificate in certificate chain) show a whole bunch of perfectly normal looking certificates much the same as the "unable to get local issuer certificate" entries. For example:
> CN=_.events.data.microsoft.com,OU=WSE,O=Microsoft,L=Redmond,ST=WA,C=US CN=settings-win.data.microsoft.com,OU=WSE,O=Microsoft,L=Redmond,ST=WA,C=US C=US,ST=California,O=Apple Inc.,CN=fmip.icloud.com CN=_.teamviewer.com
> CN=adobe.io,OU=IT,O=Adobe Systems Incorporated,L=San Jose,ST=California,C=US
> 
> Is there anything I can do to filter these out perhaps?

You may want to see the following:

https://groups.google.com/g/security-onion/c/KNJ8QY3T6_M/m/3zIryzwIAgAJ
